src/plugins/logoScreensaver/plugin.js

```diff
@@ -59,23 +59,6 @@ define(['pluginManager'], function (pluginManager) {
             return elem.animate(keyframes, timing);
         }
 
-        function shake(elem, iterations) {
-            var keyframes = [
-                { transform: 'translate3d(0, 0, 0)', offset: 0 },
-                { transform: 'translate3d(-10px, 0, 0)', offset: 0.1 },
-                { transform: 'translate3d(10px, 0, 0)', offset: 0.2 },
-                { transform: 'translate3d(-10px, 0, 0)', offset: 0.3 },
-                { transform: 'translate3d(10px, 0, 0)', offset: 0.4 },
-                { transform: 'translate3d(-10px, 0, 0)', offset: 0.5 },
-                { transform: 'translate3d(10px, 0, 0)', offset: 0.6 },
-                { transform: 'translate3d(-10px, 0, 0)', offset: 0.7 },
-                { transform: 'translate3d(10px, 0, 0)', offset: 0.8 },
-                { transform: 'translate3d(-10px, 0, 0)', offset: 0.9 },
-                { transform: 'translate3d(0, 0, 0)', offset: 1 }];
-            var timing = { duration: 900, iterations: iterations };
-            return elem.animate(keyframes, timing);
-        }
```

Comment:
This animation just isn't added to the list.

Reply:
It isn't, but it's basically moving the image very fast  from side to side, which doesn't really look all that sexy. Hence why I removed it instead of re-enabling it.